app/vminsert/netstorage/netstorage.go

```diff
@@ -743,7 +745,7 @@ func getNotReadyStorageNodeIdxs(sns []*storageNode, dst []int, snExtra *storageN
 func (sn *storageNode) trySendBuf(buf []byte, rows int) bool {
 	sent := false
 	sn.brLock.Lock()
-	if sn.isReady() && len(sn.br.buf)+len(buf) <= maxBufSizePerStorageNode {
+	if len(sn.br.buf)+len(buf) <= maxBufSizePerStorageNode {
```

Comment:
This change leads to filling of the buffer for unavailable vmstorage node. This may lead to the delay of data transfer from the buffer of unavailable vmstorage node to the remaining vmstorage nodes.